cristin-commons/src/test/java/no/unit/nva/utils/VersioningUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package no.unit.nva.utils;
+
+import static no.unit.nva.utils.VersioningUtils.extractVersion;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import org.junit.jupiter.api.Test;
+
+class VersioningUtilsTest {
+
+    public static final String ACCEPT_HEADER_VALUE = "application/json; version=2023-05-10";
```

Comment:
Add a test for `application/json; version="2023-05-10"` too. Some client library may quote parameter values in a mime type.